depends/README.usage

```diff
@@ -0,0 +1,34 @@
+To build dependencies for the current arch+OS:
+    make
+To build for another arch/OS:
+  make HOST=host-platform-triplet && make HOST=host-platform-triplet
+  (For example: make HOST=i686-w64-mingw32 -j4)
+
+A prefix will be generated that's suitable for plugging into Bitcoin's
+configure. In the above example, a dir named i686-w64-mingw32 will be
+created. To use it for Bitcoin:
+
+./configure --prefix=`pwd`/depends/i686-w64-mingw32
+
+No other options are needed, the paths are automatically configured.
+
+Dependency Options:
+The following can be set when running make: make FOO=bar
+
+SOURCES_PATH: downloaded sources will be placed here
+BASE_CACHE: built packages will be placed here
+SDK_PATH: Path where sdk's can be found (used by OSX)
+FALLBACK_DOWNLOAD_PATH: If a source file can't be fetched, try here before giving up
+NO_QT: Don't download/build/cache qt and its dependencies
+NO_WALLET: Don't download/build/cache libs needed to enable the wallet
+NO_UPNP: Don't download/build/cache packages needed for enabling upnp
+DEBUG: disable some optimizations and enable more runtime checking
+
+If some packages are not built, for example 'make NO_WALLET=1', the appropriate
+options will be passed to bitcoin's configure. In this case, --disable-wallet.
+
+Additional targets:
+download: run 'make download' to fetch all sources without building them
+download-osx: run 'make download-osx' to fetch all sources needed for osx builds
```

Comment:
Making a note that we don't support OSX and Windows yet would be a good idea. That said, on second thought, the easiest solution might be to write README.armory or something similar where we spell out any caveats and other useful info. That way, at least we have one spot we can look at for more info on what works and what doesn't.